src/plugins/custom_import_map/test/jest.config.js

```diff
@@ -14,33 +14,20 @@ module.exports = {
     modulePaths: ['node_modules', `../../node_modules`],
     coverageDirectory: './coverage',
     moduleNameMapper: {
+      '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/test/mocks/styleMock.js',
       '\\.(css|less|scss)$': '<rootDir>/test/mocks/styleMock.js',
       '^ui/(.*)': '<rootDir>/../../src/legacy/ui/public/$1/',
     },
     snapshotSerializers: ['../../node_modules/enzyme-to-json/serializer'],
     coverageReporters: ['lcov', 'text', 'cobertura'],
-    testMatch: ['**/*.test.js', '**/*.test.tsx'],
-    collectCoverageFrom: [
-      '**/*.js',
-      '!**/node_modules/**',
-      '!**/index.js',
-      '!<rootDir>/public/app.js',
-      '!<rootDir>/public/actions/**',
-      '!<rootDir>/public/components/Charts/Highlight/Highlight.js',
-      '!<rootDir>/public/reducers/**',
-      '!<rootDir>/public/store.js',
-      '!<rootDir>/test/**',
-      '!<rootDir>/server/**',
-      '!<rootDir>/coverage/**',
-      '!<rootDir>/scripts/**',
-      '!<rootDir>/build/**',
-      '!<rootDir>/gather-info.js',
-      '!<rootDir>/cypress/**',
-      '!**/vendor/**',
+    testMatch: ['**/*.test.js', '**/*.test.jsx', '**/*.test.ts', '**/*.test.tsx'],
```

Comment:
do we have test.jsx file?

Reply:
not at the moment, but the expression covers all the cases.

Reply:
Yeah my point is that I don't see a lot use case for jsx and we should also be restrict on file types that we accept.

```
In order to use JSX you must do two things.
Name your files with a .tsx extension
Enable the [jsx](https://www.typescriptlang.org/tsconfig#jsx) option
```
For all the jsx, you still need to name it with tsx extension. 

Reference:
jsx usage: https://www.typescriptlang.org/docs/handbook/jsx.html
react without jsx: https://reactjs.org/docs/react-without-jsx.html

Reply:
IMO since in this context we're looking for test files it's OK (even preferable) to cast a wider net than needed.  The failure mode of a dev writing a test file (with, say, .jsx extension) and it silently not getting run is worse than jest picking up a file that doesn't have tests inside.

You can also shorten this chunk to a single regex: "(test)\.[jt]sx?", but I get it if you don't want to :P.  I see the ".[jt]sx?" regex around pretty commonly nowadays.

Reply:
I think this is definitely not a blocker. I am okay with no change here but we still need to make sure no jsx files unless some specific reason.

Reply:
Thanks @ananzh and @CPTNB. For now since we do not have any jsx files, I'll get rid of that from the expression. Appreciate your feedback!

Reply:
Wanted to add a comment you can do a spread operator too to take same of the default configs from our repo: https://github.com/opensearch-project/security-dashboards-plugin/blob/main/test/jest.config.server.js#L16

Reply:
Good point @kavilla . I completely forget this. I think this is a clean and right thing to do.